### PR TITLE
EQ tools updates plus add new and fix some EQ topology includes

### PR DESCRIPTION
**Pull request by singalsu**
This pull request is supports @fredoh9 's work with EQ related topologies. I've fixed in generators the private data names to be similar as in his PR.

I also cleaned up and updated all three EQ generator scripts: example_iir_eq.m, example_fir_eq.m, and example_spk_eq.m.

There is need for test responses data for FIR and IIR so the topology data files are added for some effect responses those are straightforward to validate due to operating at mid frequencies and not just bass (can be measured with inexpensive setup).

While cleaning up IIR generator I found out that some merged +20 dB IIR high-pass responses were with 0 dB gain due to a bug in script. Fortunately the used filter data sets are not used so no errors should exist due to this.

This PR consists of all above changes.

### CODE REVIEW

**Comment by singalsu** [comment]
@lrgirdwo Exactly, I'm doing EQ frequency response check with testbench, ETA soon hopefully. Also THD+N check is useful. It's fast to test every provided filter coefficient set with testbench so there's need to check maybe only one one per component in real device.

File: tools/tune/eq/example_spk_eq.m [hunk 1]
```diff
@@ -134,9 +127,9 @@ function example_spk_eq()
                 assign_response, ...
                 [ bq_fir ]);
         bp_fir = eq_fir_blob_pack(bm_fir);
-        eq_alsactl_write('example_spk_eq_fir.txt', bp_fir);
-        eq_blob_write('example_spk_eq_fir.blob', bp_fir);
-	eq_tplg_write('example_spk_eq_fir.m4', bp_fir, 'FIR');
+        eq_alsactl_write(fn_fir_sofctl, bp_fir);
+        eq_blob_write(fn_fir_blob, bp_fir);
+	eq_tplg_write(fn_fir_tplg, bp_fir, fir_priv, fir_comment);
```
Comment (kv2019i) [inline tools/tune/eq/example_spk_eq.m]: @singalsu minor nti: indentation seems off here

File: tools/tune/eq/example_spk_eq.m [hunk 2]
```diff
@@ -147,9 +140,9 @@ function example_spk_eq()
                 assign_response, ...
                 [ bq_iir ]);
         bp_iir = eq_iir_blob_pack(bm_iir);
-        eq_alsactl_write('example_spk_eq_iir.txt', bp_iir);
-        eq_blob_write('example_spk_eq_iir.blob', bp_iir);
-	eq_tplg_write('example_spk_eq_iir.m4', bp_iir, 'IIR');
+        eq_alsactl_write(fn_iir_sofctl, bp_iir);
+        eq_blob_write(fn_iir_blob, bp_iir);
+	eq_tplg_write(fn_iir_tplg, bp_iir, iir_priv, iir_comment);
```
Comment (kv2019i) [inline tools/tune/eq/example_spk_eq.m]: ditto

File: tools/tune/eq/example_spk_eq.m (hunk 1 above)
Comment (singalsu) [reply tools/tune/eq/example_spk_eq.m]: Yep, there's mix of tabs and spaces and Github shows non even line starts while emacs doesn't. There would be more white space changes to fix this.

**Comment by lgirdwood** [comment]
CI known issues.
@singalsu you can always send another PR for tab/space fixes.

**Pull request merged**